docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -335,6 +335,71 @@ When I select element located `$locator` and upload file `$filePath`
 When I select element located by `id(uploadfile)` and upload file `/folder/file_for_upload.png`
 ----
 
+=== Drag and drop steps
+
+This set of steps facilitates the dragging and dropping of elements on a web page.
+Desired position relative to the target element (TOP, BOTTOM, LEFT, RIGHT, CENTER, LEFT_TOP, RIGHT_TOP, LEFT_BOTTOM, RIGHT_BOTTOM).
+
+==== The drag and drop using mouse
+
+This step for a drag-and-drop operation for specified elements.
+
+[source,gherkin]
+----
+When I drag element located by `$draggable` and drop it at $location of element located by `$target`
+----
+
+* `$draggable` - The <<_locator,locator>> to identify the element to be dragged
+* `$location` - Action to select the position state of the element (TOP, BOTTOM, LEFT, RIGHT, CENTER, LEFT_TOP, RIGHT_TOP, LEFT_BOTTOM, RIGHT_BOTTOM).
```

Comment:
it's not really action, but rather target position relatively to the target element